PulsePace/Gameplay/Commands/Command.swift

```diff
@@ -12,11 +12,11 @@ protocol Command {
     associatedtype ActionArguments
     typealias Action = (ActionArguments) -> Void
     var action: Action { get }
-    func execute(inputData: ActionArguments)
+    func onChanged(inputData: ActionArguments)
```

Comment:
I think this can stay as `execute()` itself, because `Command` is more of a general wrapper class for `Action`s and it wont make sense to say the command 'changed'?

So i had added `executeOnEnded()` to be specific to `InputCommand`s, because we'll probably have an action to execute on when input ends as well.

Reply:
I think I accidentally made this change when I was refactoring 😓we can revert this change. am thinking now though, could it be clearer if we name it as `executeAction` and `executeCompletion`? since `InputCommand` has `action` and `completion`.